test/server-side-rendering/ssr-1.test.js

```diff
@@ -14,7 +14,9 @@ import {
 	try_load_json,
 	try_read_file
 } from '../helpers.js';
-import { assert_html_equal } from '../html_equal.js';
+import { setup_html_equal } from '../html_equal.js';
+
+const { htmlEqualWithOptions } = setup_html_equal();
```

Comment:
wouldn't this need to be `html_equal_with_options` since it's our own code

Reply:
Ahh, there are a quite a few test files that use these in the camelCase form, but checking again I don't think they directly import it so should be fine to camelCase them where they are passed.